scripts/gen-aliasarr.template.sh

```
@@ -837,148 +837,77 @@ _arr_gluetun_key() {
+_arr_port_guard_events_file() {
+  printf '%s/port-guard-events.log' "$(_arr_port_guard_state_dir)"
 }
 
-_arr_pf_pm_script() {
-  local candidate
-  candidate="${ARR_STACK_DIR}/scripts/vpn-port-watch.sh"
-  if [ -x "$candidate" ]; then
-    printf '%s' "$candidate"
-    return 0
+_arr_port_guard_print_json() {
+  local file="$(_arr_port_guard_status_file)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)